### DISCUSSION TITLE
prefixURL not working correctly when used as environmental variable

### DISCUSSION THREAD

**Discussion by DanielKlys7** [sindresorhus/ky#401] [Q&A] [answered]
Working on `localhost:3000`, `.env` contains `REACT_APP_API_URL=​https://www.reddit.com/`

Whenever we want to make a request using this client:

```
export const client = ky.create({
  prefixUrl: process.env.REACT_APP_API_URL,
});
```
it is made to `http://localhost:3000/%E2%80%8Bhttps://www.reddit.com/r/music/new.json 404 (Not Found)`

if `prefixUrl` is set to api directly in `create` options parameter, everything works fine.

**Comment by sindresorhus** [comment, marked as answer]
> if prefixUrl is set to api directly in create options parameter, everything works fine.

I think this is a pretty clear sign that the issue is not with Ky.

---

Looks like the environment variable value includes some hidden spaces.

Try `process.env.REACT_APP_API_URL` => `process.env.REACT_APP_API_URL.trim()`

**Comment by sholladay** [comment]
Yeah, definitely a problem with the way the env var is being parsed and serialized.

Log the env var and check for any unexpected characters:

```js
console.log(process.env.REACT_APP_API_URL)
```

Or:

```js
console.log(process.env.REACT_APP_API_URL === '<your url here>')
```
